### PR TITLE
feature:add unpause command

### PR DESCRIPTION
**Pull request by mozhuli**
**1.Describe what this PR did**

add unpause command

**2.Does this pull request fix one issue?** 

**3.Describe how you did it**

**4.Describe how to verify it**

**5.Special notes for reviews**

### CODE REVIEW

**Comment by allencloud** [comment]
Sounds good to me. @mozhuli 
Thanks a lot for your contribution. 
While I am wondering if you can add some test cases in cli integration test. With test, I think the code added is more guaranteed.

**Comment by ZouRui89** [comment]
Thanks a lot for your contribution. 😄@mozhuli It would be better if you could sign off your commits. 
#git config user.name "YourName" 
#git config user.email "YourEmailAddress"
Then use 
#git commit -s -m "xxxxxx" to commit and push back to github.

**Comment by mozhuli** [comment]
@allencloud @ZouRui89 updated!

File: apis/server/container_bridge.go [hunk 1]
```diff
@@ -164,6 +164,18 @@ func (s *Server) pauseContainer(ctx context.Context, resp http.ResponseWriter, r
 	return nil
 }
 
+func (s *Server) unpauseContainer(ctx context.Context, resp http.ResponseWriter, req *http.Request) error {
+
```
Comment (skoo87) [inline apis/server/container_bridge.go]: remove the blank line.

File: ctrd/container.go [hunk 2]
```diff
@@ -257,6 +257,30 @@ func (c *Client) PauseContainer(ctx context.Context, id string) error {
...
+	if err := pack.task.Resume(ctx); err != nil {
+		if !errdefs.IsNotFound(err) {
+			return errors.Wrap(err, "failed to resume task")
+		}
+	}
+
+	logrus.Infof("success to unpause container: %s", id)
+
+	return nil
+
```
Comment (skoo87) [inline ctrd/container.go]: delete the blank line.

File: apis/server/container_bridge.go (hunk 1 above)
Comment (mozhuli) [reply apis/server/container_bridge.go]: updated

**Comment by ZouRui89** [comment]
LGTM. Thanks a lot for the great code implementation and test cases. We are merging it.😄

**Pull request merged**